vendor/engines/projects/app/models/projects/facility_extension.rb

```diff
@@ -0,0 +1,17 @@
+module Projects
+
+  module FacilityExtension
+
+    extend ActiveSupport::Concern
+
+    included do
+      has_many :projects,
+               class_name: "Projects::Project",
+               foreign_key: :facility_id,
+               inverse_of: :facility,
+               dependent: :destroy # Though Facilities cannot be destroyed
```

Comment:
Facilities can't be destroyed yet, but if that ever changes, we'll be ready. We have similar `dependent: :destroy` relations for `order_imports` and `user_roles`.